Bump configparser dependency back to 3 again.

Upstream lowered this dependency (for the benefit of Fedora?) as part of
https://github.com/AsahiLinux/speakersafetyd/commit/063db4854bb02927a1b24b8bf3577f51b2c305a4#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542L11

As debian has the newer configparser crate packaged, upgrade it again.

Index: speakersafetyd/Cargo.toml
===================================================================
--- speakersafetyd.orig/Cargo.toml
+++ speakersafetyd/Cargo.toml
@@ -32,7 +32,8 @@ features = ["derive"]
 version = "2.0.0"
 
 [dependencies.configparser]
-version = "^2.1.0"
+version = "^3.0.2"
+features = ["indexmap"]
 
 [dependencies.json]
 version = "0.12.4"
